.vib/magento/cypress/cypress/integration/magento_spec.js

```diff
@@ -0,0 +1,103 @@
+/// <reference types="cypress" />
+import { random, allowDataUsage } from './utils';
+
+before(() => {
+  cy.login();
+  allowDataUsage();
+});
+
+it('allows admin to add a product to the store', () => {
+  cy.get('#menu-magento-catalog-catalog').click();
+  cy.get('.item-catalog-products').click();
+  cy.contains('Salable Quantity');
+  cy.get('#add_new_product-button').click();
+  cy.fixture('products.json').then((product) => {
+    cy.get('[name="product[name]"]').type(
+      `${product.newProduct.productName}.${random}`
+    );
+    cy.get('[name="product[price]"]').type(product.newProduct.price);
+  });
+  cy.get('#save-button').click();
+  cy.contains('You saved the product');
+  cy.get('#menu-magento-catalog-catalog').click();
+  cy.get('.item-catalog-products').click();
+  cy.fixture('products.json').then((product) => {
+    cy.contains(`${product.newProduct.productName}.${random}`);
+  });
+});
+
+it('allows user to create a customer account', () => {
+  cy.visit('/');
+  cy.contains('Create an Account').click();
+  cy.fixture('customers.json').then((customer) => {
+    cy.get('.field-name-firstname').type(
+      `${customer.newCustomer.firstName}.${random}`
+    );
+    cy.get('#lastname').type(`${customer.newCustomer.lastName}.${random}`);
+    cy.get('#email_address').type(`${random}.${customer.newCustomer.email}`);
+    cy.get('#password').type(customer.newCustomer.password);
+    cy.get('#password-confirmation').type(customer.newCustomer.password);
+  });
+  cy.contains('button', 'Create an Account').click();
+  cy.contains('Thank you for registering with Main Website Store.');
```

Comment:
I would reduce this to:

```suggestion
  cy.contains('Thank you for registering');
```

which is enough to now registration completed successfully.

Reply:
Indeed!